.goreleaser.yml

```diff
@@ -17,3 +17,28 @@ release:
   # If set to true, will mark the release as not ready for production.
   prerelease: auto
 
+brew:
+  name: ld-find-code-refs
+
+  description: "Enables you to keep track of feature flag usage in your code from LaunchDarkly."
```

Comment:
should we standardize the description? I actually like this description better, but it would be nice to make it consistent across the brew, docker images, orb, and github description. At least the orb description could be updated in this pr:
https://github.com/launchdarkly/ld-find-code-refs/blob/f5c42f8e21a5e2c088cca555d952d787753d6f4a/build/package/circleci/orb.yml#L2

Reply:
ok, for now I will standardize to the orb description.